alica_engine/src/engine/parser/ModelFactory.cpp

```diff
@@ -1419,8 +1433,8 @@ const EntryPoint* ModelFactory::generateIdleEntryPoint()
     EntryPoint* idleEP = new EntryPoint();
     idleEP->setName("IDLE-ep");
     idleEP->setId(EntryPoint::IDLEID);
-    idleEP->setMinCardinality(0);
-    idleEP->setMaxCardinality(std::numeric_limits<int>::max());
+    idleEP->_index = -42;
```

Comment:
negative meaning of life, the universe, and everything?

Reply:
Just a hint that this is a special EP. Though we might be able to get rid of it entirely soon.